app/models/concerns/rad_user.rb

```diff
@@ -100,12 +100,8 @@ def to_s
     end
   end
 
-  def active
```

Comment:
removed the `active` method, we've been using `active?` instead going forward and just want to avoid confusion and have a method that is easier to search for when the time comes to refactor anything